docs/how_to/cluster_hard_way.md

```diff
@@ -183,7 +204,17 @@ Shortly after, you should see your node complete bootstrapping:
 20:10:14.764771[I] successfully updated topology to 3 hosts
 ```
 
-Read more about namespaces and the various knobs in the docs.
+Below are recommendations for block size based on the resolution and retention of the namespace:
+
+| Retention / Resolution | 10s | 1m  | 10m | 1h  |
+|------------------------|-----|-----|-----|-----|
+| 12h                    | 30m | 2h  | N/A | N/A |
+| 24h                    | 1h  | 4h  | N/A | N/A |
+| 168h                   | 6h  | 12h | 24h | 48h |
+| 720h                   | 24h | 24h | 48h | 96h |
+| 8760h                  | 24h | 24h | 48h | 96h |
```

Comment:
Are these 1-1 the same as what we have created here?
https://github.com/m3db/m3/blob/master/src/query/api/v1/handler/database/create.go#L95-L116

It would be good to make sure they are 1-1 or else we're kind of telling people different advice depending on whether they're creating themselves or using the `database/create` API.